src/api/spec/jobs/update_backend_infos_spec.rb

```diff
@@ -15,10 +15,10 @@
 
   context "properly set" do
     let(:other_package) { create(:package, name: 'OtherFakePackage') }
-    let(:backend_package) { BackendPackage.create(package: other_package, links_to: package) }
 
     before do
-      allow(subject).to receive(:update_package)
+      allow_any_instance_of(Package).to receive(:update_backendinfo)
+      BackendPackage.create(package: other_package, links_to: package)
```

Comment:
@mdeniz @Ana06 The code now touches the 2 lines that were not covered before, but does it test it? I would at least test that linked package get's updated as well.

Also allowing any instance of package to be called with update_backendinfo might be a bit to unspecific. Can we instead set the expectations to two packages (original and the linked one)?